test-sbt/shared/src/main/scala/zio/test/runner/ZTestRunner.scala

```diff
@@ -106,18 +101,32 @@ object ZTestEvent {
       0L
     )
 
-  private def maybeThrowable(testResult: ZTestResult) = testResult match {
-    case ZTestResult.Failure(FailureDetails.Runtime(cause), _) =>
+  private def maybeThrowable(testResult: TestResult) = testResult match {
+    case Assertion.Failure(FailureDetails.Runtime(cause)) =>
```

Comment:
The full details of the failure would be nice to print out at some point, pretty print, if not here then elsewhere (I suppose in the reporter).